awscrt/s3.py

```diff
@@ -284,7 +295,8 @@ def on_shutdown():
             throughput_target_gbps,
             enable_s3express,
             memory_limit,
-            s3_client_core)
+            s3_client_core,
+            network_interface_names)
```

Comment:
trivial/style: `s3_client_core` is kind of special, it's the only non-option-from-user thing passed along here.
Either keep it as the final arg, or move it to be the first arg so people aren't tempted to keep appending new args to the end of this list

Reply:
Thanks, updated.